atomics/include/desul/atomics/Atomic_Ref.hpp

```diff
@@ -10,13 +10,12 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #define DESUL_ATOMIC_REF_IMPL_HPP_
 
 #include <cstddef>
+#include <desul/atomics/Common.hpp>
+#include <desul/atomics/Generic.hpp>
+#include <desul/atomics/Macros.hpp>
```

Comment:
Why the change to intermixing system and project includes, is this Kokkos style?

Reply:
You mean intermixing including with double quotes and angle brackets?

Reply:
I mean mixing system libraries, parts of std for example, with parts of desul. I also notice the switch from quotes to angle brackets, but since we were using the full path rather than relative for includes that's more correct anyway unless we want to update them to use relative paths.

Reply:
I tend to prefer angle brackets.  The reordering was clang-format and I also dislike it very much.  We can rework it.
I am not opposed to relative path with double-quote.  Do you want me to change?

Reply:
Nah, fully qualified with angle brackets is fine.  I would say change the clang-format though, I tend to like these options (pseudo-syntax, I don't have a working file in front of me):

```
IncludeBlocks=Regroup
IncludeCategories:
  - Regex:           '^["<]desul/'
    Priority:        2
    CaseSensitive:   true
  - Regex:           '<[[:alnum:].]+>'
    Priority:        3
  - Regex:           '.*'
    Priority:        1
    SortPriority:    0
```

IIRC that orders unknown includes first, then desul stuff, then angle-bracket searched headers that aren't from desul last.  I'd be good with adjusting that order, but I tend to like to have project headers followed by system ones, less chance of accidental dependency.

Reply:
Any objection to fix as a follow up instead?

Reply:
None, I'll create an issue from the above comment and resolve this.